### ISSUE TITLE
2.3 less accurate than 2.2.x

### ISSUE THREAD

**Issue by viaregio** [explosion/spaCy#5662]
## How to reproduce the behaviour
I trained a model with the following command:
```bash
spacy train -b de_core_news_md -p ner -R de my-model train.json dev.json
```
The files `train.json` and `dev.json` contain both 1000 lines of tagged text. With version 2.2.4 of spacy I got over 98% of precision and recall. With version 2.3 I only get around 95% of precision and recall. I repeated the process for both versions multiple times, but the difference kept roughly the same, about 3%.

Do I need another call of `spacy train` to get the same precision and recall levels?

## Info about spaCy

* **spaCy version:** 2.2.4
* **Platform:** macOS-10.15.5-x86_64-i386-64bit
* **Python version:** 3.8.3

**Comment by adrianeboyd** [comment]
The vectors were modified from  v2.2 to v2.3 and it looks like the previous vectors worked better for your task.

For our internal data (WikiNER), the updated vectors led to better performance, probably in part because Wikipedia was one of text sources for the new vectors. The previous German vectors seemed to have problems related to coverage (there were many frequent words without vectors), so I'd warn against using them in general, but there's no reason not to keep using for your task if you want.

If you just want to train an NER model, I can help you use `spacy train` for v2.3 with the v2.2 vectors, but it looks like you want to keep the tagger and parser from `de_core_web_md`. Since the tagger and parser are trained using the new vectors and the model can only support one set of vectors, there's no good way to have the tagger and parser use the new vectors and the NER model use the old vectors.

**Comment by viaregio** [comment]
@adrianeboyd Thank you very much for your offer. With the following command 
```bash
spacy train -p ner de model-5 train.json dev.json
```
I get values for precision and recall aprox. 0.1% lower than with the previous command using the model de_core_news_md. So with version 2.3 the model doesn't seem to make a difference. If I remember correctly in version 2.2.4 I got better results starting from the model de_core_news_md than by starting with the blank model. I'm not interested in the results of parser and tagger, but I thought their results might help with the entity recognition (Maybe this is a naive assumption, but I just started playing around with NLP and spacy). I have no training data for tagger and parser, so I kept the existing tagger and parser.

How could I use the 2.2 vectors with the version 2.3 of spacy? I would give it a try.

**Comment by adrianeboyd** [comment]
The three core components are completely independent, so the tagger and parser don't affect the NER results.

If you want to use the v2.2 vectors, you can provide the vectors from a v2.2 model with `-v`. If you have the model installed in a virtual environment, it will look something like this:

```
spacy train de model train dev -p ner -v /path/to/venv/lib/python3.7/site-packages/de_core_news_md/de_core_news_md-2.2.5
```

Or you can just download the model `.tar.gz` from the release page (e.g., https://github.com/explosion/spacy-models/releases/de_core_news_md-2.2.5), unpack it, and load the vectors from the model directory two levels down inside the package (`de_core_news_md-2.2.5/de_core_news_md/de_core_news_md-2.2.5`).

There will be a warning that the model isn't compatible, but this shouldn't affect the vectors. (I haven't tested this carefully, so do make sure the vectors / results end up as you expect.) After training you'll have a v2.3 model with those vectors, which you can use with `-v` without warnings in the future.

**Comment by viaregio** [comment]
Thank you very much. I will give it a try.

**Comment by viaregio** [comment]
This didn't work. I got the following traceback:

```bash
Traceback (most recent call last):
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/__main__.py", line 33, in <module>
    plac.call(commands[command], sys.argv[1:])
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/plac_core.py", line 367, in call
    cmd, result = parser.consume(arglist)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/plac_core.py", line 232, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/cli/train.py", line 266, in train
    _load_vectors(nlp, vectors)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/cli/train.py", line 645, in _load_vectors
    util.load_model(vectors, vocab=nlp.vocab)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/util.py", line 172, in load_model
    return load_model_from_path(Path(name), **overrides)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/util.py", line 216, in load_model_from_path
    component = nlp.create_pipe(factory, config=config)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/language.py", line 309, in create_pipe
    return factory(self, **config)
  File "/Users/uli/.pyenv/versions/3.8.3/lib/python3.8/site-packages/spacy/language.py", line 1080, in factory
    return obj.from_nlp(nlp, **cfg)
  File "pipes.pyx", line 62, in spacy.pipeline.pipes.Pipe.from_nlp
  File "pipes.pyx", line 378, in spacy.pipeline.pipes.Tagger.__init__
TypeError: __init__() got multiple values for keyword argument 'vocab'
```
I think this is related to #5620.

**Comment by adrianeboyd** [comment]
Yes, this will be fixed in v2.3.1.
